Remove unused type ignores and casts (#1031)
aio-libs/yarl: CHANGES/1031.contrib.rst

> Update mypy, remove unused ignores and casts

Reviewer: Don't use imperative mood. Change notes have different style. Optionally, add a byline. Add a trailing period to the sentence. Use proper capitalization for MyPy.

> Update mypy, remove unused ignores and casts

Reviewer 2: ```suggestion
Updated mypy and removed unused ignores and casts -- by :user:`hauntsaninja`.
```

> Updated mypy and removed unused ignores and casts -- by :user:`hauntsaninja`.

Reviewer 2: ```suggestion
Updated MyPy and removed unused ignores and casts -- by :user:`hauntsaninja`.
```

> Updated MyPy and removed unused ignores and casts -- by :user:`hauntsaninja`.

Reviewer 2: ```suggestion
Removed unused type ignores and casts -- by :user:`hauntsaninja`.
```
